use-cases.md

```diff
@@ -180,7 +180,8 @@ The client and server examples above propagated the Span/Trace over the wire, in
     span.set_trace_attribute('auth-token', '.....')
 
     # server side (one or more levels down from the client)
-    token = span.get_trace_attribute('auth-token')
+    h_ctx, h_attr = tracer.propagate_span_as_text(span)
+    token = h_attr.get_trace_attribute('auth-token')
```

Comment:
Also, this won’t work given current semantics, because the OT tracer has no obligation to return the key/value map with any specific keys. For example, it can merge all trace attributes into a single base64 string and return a single key in the map. Or, more likely, it can prepend a certain prefix to all attribute keys so that the map keys are not mixed up with any other random HTTP headers, as well as to allow the tracer on the receiving side to read only the headers that represent trace attributes.